TUM Campus App/PlansTableViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  PlansTableViewController.swift
+//  TUM Campus App
+//
+//  Created by Florian Gareis on 16.12.16.
+//  Copyright © 2016 LS1 TUM. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+
+class PlansTableViewController: UITableViewController {
+    
+    // MARK: - Table view data source
+    lazy var plans = Plans()
```

Comment:
Thank you very much for your patience! Now I think I understand the getter and setter thing :)